spec/features/course/assessment/assessment_attempt_spec.rb

```diff
@@ -137,6 +137,15 @@
         # Test that ACE Editor has initialised correctly and the content is shown.
         expect(page).to have_selector('div.ace_editor')
         expect(find('.tab-pane.active')['id']).to eq(question_id.to_s)
+
+        # Test that the step parameter when editing submissions goes straight to the question
+        # Switch back to the first question to test this.
+        submission = Course::Assessment::Submission.
+                     where(assessment: assessment_tabbed, creator: user).first
+        first_question_id = assessment_tabbed.questions.first.id
+
+        visit edit_course_assessment_submission_path(course, assessment_tabbed, submission, step: 1)
```

Comment:
I thought of that, but the currently loaded tab is step 2, so if nothing happens the test's purpose is also circumvented.